quiche/src/path.rs

```diff
@@ -430,12 +431,10 @@ impl Path {
     pub fn on_loss_detection_timeout(
         &mut self, handshake_status: HandshakeStatus, now: time::Instant,
         is_server: bool, trace_id: &str,
-    ) -> (usize, usize) {
-        let (lost_packets, lost_bytes) = self.recovery.on_loss_detection_timeout(
-            handshake_status,
-            now,
-            trace_id,
-        );
+    ) -> OnLossDetectionTimeoutOutcome {
+        let on_loss_detection_timeout_outcome = self
```

Comment:
I would consider: `on_loss_detection_timeout_outcome` -> `outcome`

but this seems fine.